<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>681</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="capacity">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>capacity</string>
    </property>
   </widget>
   <widget class="QPushButton" name="size">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>130</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>size</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_el">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>push_back</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>180</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="del_el">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>pop_back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>230</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="insert">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>insert</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="num_insert">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>280</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>290</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;-num of insert el</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="val_insert">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>320</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>330</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;-val of insert el</string>
    </property>
   </widget>
   <widget class="QPushButton" name="empty">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>empty?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="erase">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>420</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>erase</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="num_erase">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>420</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>430</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;-num of erase el</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
